x/accounts/defaults/multisig/account_test.go

```diff
@@ -574,7 +596,7 @@ func TestProposalPassing(t *testing.T) {
 	require.NoError(t, err)
 	for _, v := range cfg.Members {
 		if v.Address == "addr1" {
-			require.Equal(t, uint64(500), v.Weight)
+			require.Equal(t, 500, int(v.Weight))
```

Comment:
personal preference: this is an existing test but it would be great to confirm that the members list is exactly what it should be. Something like `assert.Equals(t, expMembers, cfg.Members)` for example gives confidence that no other members are dropped or modified or duplicates exists. 
if you want to keep the current version, you can add a break or return to exit early